docs/install.md

```diff
@@ -1,3 +1,10 @@
+<!--
+---
+linkTitle: "Installation"
+weight: 1
```

Comment:
"1" is already used by `tasks.md` - so maybe we can change that two 2?
There are already some details on the install process in https://tekton.dev/docs/getting-started/#installation - those are in the website repo though... so maybe we could add a link in there to this document too?

Reply:
I'm glad you noticed! At first, I used `1` for `install.md` and bumped all the other weights. But the thing is, there are multiple documents with with `2` already. Seeing this, I took the lazy approach of using `1` for both `tasks.md` and `install.md`, figuring they would be sorted alphabetically (as seems to be the case for the `2`s). But I can just as well take another pass and make sure weights are unique.

Reply:
As to the getting-started page: sure! Something like:

> For more detailed installation instructions, see [Installation](https://tekton.dev/docs/pipelines/install/).

added to the bottom of the Installation section?

Reply:
Here's a stab at a new set of weights. I tried ordering core components at the top, but I'm no Tekton expert yet, so let me know if something seems out of place. 
```
install.md                                      weight: 1
tasks.md                                        weight: 2
taskruns.md                                     weight: 3
pipelines.md                                    weight: 4
pipelineruns.md                                 weight: 5
resources.md                                    weight: 6
workspaces.md                                   weight: 7
events.md                                       weight: 8
runs.md                                         weight: 9
auth.md                                         weight: 10
container-contract.md                           weight: 11
tekton-bundle-contracts.md                      weight: 12
logs.md                                         weight: 13
labels.md                                       weight: 14
conditions.md                                   weight: 15
podtemplates.md                                 weight: 16
enabling-ha.md                                  weight: 17
metrics.md                                      weight: 18
variables.md                                    weight: 19
tekton-controller-performance-configuration.md  weight: 20
migrating-from-knative-build.md                 weight: 40
migrating-v1alpha1-to-v1beta1.md                weight: 40
deprecations.md                                 weight: 50

```
I'm bumping the weight values for deprecations and migration guides (so they will stay at the bottom should any new docs be added in the future).

Reply:
What do you think about this suggestion @afrittoli? I suppose pushing an update will require new test runs, so.. should I go ahead with this?

Reply:
@tektoncd/core-maintainers

Reply:
Suggest pushing `resources.md` down to the 15-20 range and pulling `variables.md` up to the 10-15 range since PipelineResources are deprecated and variables are used everywhere. WDYT @afrittoli ?

Also: it might make future updates easier if the weights are wider apart (e.g. 100, 200, ..., 1000, 1100)? We wouldn't need to re-shuffle when a new addition is made between two existing entries, it could just use the mid-weight between two existing documents (e.g. `100`, `150` (new), `200`).

But I don't feel strongly about either of these - lgtm otherwise, cheers!

Reply:
Good idea! Pushed a new commit applying you suggestions.

<details>

<summary>New weights</summary>

```
install.md                                      weight: 100
tasks.md                                        weight: 200
taskruns.md                                     weight: 300
pipelines.md                                    weight: 400
pipelineruns.md                                 weight: 500
workspaces.md                                   weight: 600
events.md                                       weight: 700
runs.md                                         weight: 800
variables.md                                    weight: 900 // higher up
auth.md                                         weight: 1000
logs.md                                         weight: 1100 // logs and metrics together
metrics.md                                      weight: 1200
labels.md                                       weight: 1300
podtemplates.md                                 weight: 1400
enabling-ha.md                                  weight: 1500
tekton-controller-performance-configuration.md  weight: 1600
container-contract.md                           weight: 1700 // contracts shifted downward
tekton-bundle-contracts.md                      weight: 1800
resources.md                                    weight: 2000 // deprecated, further down
conditions.md                                   weight: 2100 // deprecated, further down
migrating-v1alpha1-to-v1beta1.md                weight: 4000
migrating-from-knative-build.md                 weight: 4100
deprecations.md                                 weight: 5000
```

</details>